js/os_test.ts

```diff
@@ -183,3 +183,24 @@ test(function makeTempDirSyncPerm() {
   assert(err);
   assertEqual(err.name, "deno.PermissionDenied");
 });
+
+testPerm({ write: true }, function mkdirSync() {
+  const path = deno.makeTempDirSync() + "/dir/subdir";
+  deno.mkdirSync(path);
+  const pathInfo = deno.statSync(path);
+  assert(pathInfo.isDirectory());
+});
+
+testPerm({ write: false }, function mkdDirSyncPerm() {
+  let err;
+  try {
+    const path = deno.makeTempDirSync() + "/dir/subdir";
```

Comment:
Remove this line - currently the test is only testing the permissions of makeTempDirSync